frontend/src/components/Map/index.tsx

```diff
@@ -12,14 +13,17 @@ function Map() {
   //   });
   //   return null;
   // }
+
   return (
-    <MapContainer
-      center={[50.447731, 30.542721]}
-      zoom={9}
-      style={{ height: '100vh' }}
-    >
-      <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
-    </MapContainer>
+    <Box>
+      <MapContainer
+        center={[50.447731, 30.542721]}
+        zoom={9}
+        style={{ height: '100vh' }}
```

Comment:
try to not use inline styles (especially if it's not a __sx__). In your case, you can style parent <Box> and inside this box, add style for his children (or whatever other implementation).

Reply:
I didn't write it. This code was already there before me.